specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -153,10 +161,39 @@ Consider at least the below topics but you do not need to cover those that are n
       * Is it logged?
   * What secrets are needed by the components? How are these secrets secured and attained?
 -->
-### CICD
+### CI/CD
 
-#### e2e test image
-To run e2e tests alot of dependencies are needed (browsers, build systems etc...).
+CI/CD for frontend components leverages existing CICD for the VDK monorepo using
+GitLab.
+
+- [GitLab config for frontend components](/projects/frontend/cicd/.gitlab-ci.yml)
+- [@versatiledatakit/shared build script](/projects/frontend/cicd/build_shared.sh)
+- [@versatiledatakit/shared release script](/projects/frontend/cicd/publish_shared.sh)
+
+#### Dependency management
+
+Peer dependency versions are lower-bound, i.e. the version of a specific peer
```

Comment:
> Our main goals when it comes to dependency management are as follows:
> 
> 1. Keep peer dependencies up to date to avoid vulnerabilities
> 2. Detect problems related to peer dependencies early
> 
> Therefore, peer dependency versions are lower-bound, i.e. the version of a specific peer
> dependency should be greater than or equal to a specific number. Peer dependency
> versions should not be upper-bound unless absolutely necessary.
> 
> `package-lock.json` is not included in version control for either package.
> Instead, for each pipeline run, `npm install` is run and `package-lock.json` is
> output as an artefact at the end. Artefact retention is 7 days. This setup
> ensures that the latest dependencies are pulled for every build. It also ensures
> compatibility issues are detected early and are traceable.

Reply:
Not sure what alternatives we can even consider here. I guess we can commit the package-lock file and keep dependency versions pinned. If we agree the goals I listed are correct, then keeping dependency management the way it was originally does not comply with the goals.